.circleci/config.yml

```diff
@@ -1,6 +1,6 @@
 version: 2.1
 
-supported-eslint-versions: &supported-eslint-versions ["local", "9"]
+supported-eslint-versions: &supported-eslint-versions ["local"]
```

Comment:
Let's remove the `supported-eslint-versions` system entirely from this file. It's no longer needed.

Reply:
I left it in just incase there are new major versions of eslint that don't require any changes from us except testing that it works. wdyt?